Patch to use setuptools to allow --single-version-externally-managed

Index: setup.py
--- setup.py.orig
+++ setup.py
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 # setup.py generated by flit for tools that don't yet use PEP 517
 
-from distutils.core import setup
+from setuptools import setup
 
 packages = \
 ['testpath']
